mods/default/nodes.lua

```diff
@@ -1388,11 +1388,13 @@ minetest.register_node("default:chest", {
 	end,
     on_metadata_inventory_put = function(pos, listname, index, stack, player)
 		minetest.log("action", player:get_player_name() ..
-			" moves stuff to chest at " .. minetest.pos_to_string(pos))
+			" moves " .. stack:get_name() .. " " .. stack:get_count() ..
```

Comment:
Wouldn't it be better to use [`stack:to_string()`](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L2733)?